lib/src/parse/stylesheet.dart

```diff
@@ -1135,10 +1139,33 @@ abstract class StylesheetParser extends Parser {
     return url.startsWith("http://") || url.startsWith("https://");
   }
 
-  /// Consumes a supports condition and/or a media query after an `@import`.
+  /// Consumes a supports condition and/or a media query and/or a layer after an `@import`.
   ///
   /// Returns `null` if neither type of query can be found.
-  Tuple2<SupportsCondition?, Interpolation?>? tryImportQueries() {
+  Tuple3<SupportsCondition?, Interpolation?, Interpolation?>?
```

Comment:
here I added the layer as third item in the tuple, to reduce the diff in places using this function. But I'm wondering whether the layer should come first instead, to be in the same order than in CSS.

Reply:
@nex3 any preference about the order here ?